core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -129,9 +139,20 @@ public void cancelled(Context context) {
     return this;
   }
 
+  ClientCallImpl<ReqT, RespT> setCompressorRegistry(CompressorRegistry compressorRegistry) {
+    this.compressorRegistry = compressorRegistry;
+    return this;
+  }
+
+  ClientCallImpl<ReqT, RespT> setKnownMessageEncodings(Set<String> knownMessageEncodings) {
```

Comment:
I still prefer calling it a registry, which puts the focus on the container instead of the data in it. It's also in line with `setCompressorRegistry()` et al.